### PR TITLE
feat: update lighting and mats

### PR DESCRIPTION
**Pull request by mozhoku**
## Description

Updates scene lighting and some materials to make it similar to HDRP version. Adjusted traffic light and vehicle light scripts to work properly with the URP. These changes should fix the traffic light detection problem.

Also updated the Nishishinjuku_URP.unitypackage [here](https://drive.google.com/drive/folders/1Q6PolpCIwiVRdPNMjSBAQ5t2TcCoY-KZ).  Updated documentation with the new package link.

Enabled auto light baking. Changed default lighting settings for faster light baking. This change is intended to aid users with low-end PCs. However, those with high-end systems retain the flexibility to customize their settings as desired.

### Related links

- https://github.com/autowarefoundation/AWSIM/issues/55

## Tests performed

Tested visual changes on Windows and Linux. Didn't tested with Autoware.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by xmfcx** [comment]
Could you share the before and after images?

**Comment by mozhoku** [comment]
**HDRP** vs **Old URP** vs **New URP**
vs

**Comment by xmfcx** [comment]
Did you test this with the Autoware to see the green light performance?

**Comment by xmfcx** [comment]
| URP                                                                                                                                      | HDRP                                                                                                                                     |
|------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
| Screenshot from 2024-04-22 16-35-16 | Screenshot from 2024-04-22 16-35-20 |
| Has white                                                                                                                                | Only has the color                                                                                                                       |
| Has specular surface                                                                                                                     | Has matte or pure emissive surface                                                                                                       |

Could you fix these?

**Comment by xmfcx** [comment]
### Comparison video between AWSIM 1.2.1 with HDRP vs AWSIM Labs with URP

[Comparison  Video](https://www.youtube.com/watch?v=TAKSOZy7r2w)

**Comment by mozhoku** [comment]
I've updated the Nishishinjuku package and the post process settings. If you are going to test again you need to download the package from [here.](https://drive.google.com/drive/folders/1Q6PolpCIwiVRdPNMjSBAQ5t2TcCoY-KZ) Greens should be OK now.

image

File: docs/GettingStarted/SetupUnityProject/index.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ To properly run and use AWSIM project in Unity it is required to download map pa
 
 1. Download and import `Nishishinjuku_urp.unitypackage`
 
-    [Download Map files (unitypackage)](https://drive.google.com/drive/folders/15D5s2m3A7_wtCPio8ewRy0wL_xQtmqss){.md-button .md-button--primary}
+    [Download Map files (unitypackage)](https://drive.google.com/drive/folders/1Q6PolpCIwiVRdPNMjSBAQ5t2TcCoY-KZ){.md-button .md-button--primary}
```
Comment (xmfcx) [inline docs/GettingStarted/SetupUnityProject/index.md]: Could you assign a version to this file?

You can name it `nishishinjuku_urp_v0.1.0.unitypackage` for this version.

Within a PR, for each change, increase the PATCH number.

For each PR, increase the MINOR number.

( https://semver.org/ )

Comment (mozhoku) [reply docs/GettingStarted/SetupUnityProject/index.md]: Updated the package name in the Google Drive. Do I also need to update it in the docs?

**Comment by mozhoku** [comment]
Here is the video comparison with the updated environment:
https://youtu.be/7efsNXH0kA0

File: docs/GettingStarted/SetupUnityProject/index.md (hunk 1 above)
Comment (mozhoku) [reply docs/GettingStarted/SetupUnityProject/index.md]: Committed here: https://github.com/autowarefoundation/AWSIM/pull/56/commits/bf17e5415479e032d798b57c9e390d928bf0d489

**Review by xmfcx — APPROVED** [review]

**Comment by xmfcx** [comment]
Thanks, it works even better with the traffic lights than the original because of the lack of fog I think!

**Pull request merged**